command/assets/connect.nomad

```diff
@@ -0,0 +1,467 @@
+    # value  = "us-west1"
+
+
+    # weight can be used to indicate relative preference
+    # when the job has more than one affinity. It defaults to 50 if not set.
+    # weight = 100
+    #  }
+
+
+    # The "spread" stanza allows operators to increase the failure tolerance of
+    # their applications by specifying a node attribute that allocations
+    # should be spread over.
+    #
+    # For more information and examples on the "spread" stanza, please
+    # see the online documentation at:
+    #
+    #     https://www.nomadproject.io/docs/job-specification/spread.html
+    #
+    # spread {
+    # attribute specifies the name of a node attribute or metadata
+    # attribute = "${node.datacenter}"
+
+
+    # targets can be used to define desired percentages of allocations
+    # for each targeted attribute value.
+    #
+    #   target "us-east1" {
+    #     percent = 60
+    #   }
+    #   target "us-west1" {
+    #     percent = 40
+    #   }
+    #  }
+
+    # The "network" stanza for a group creates a network namespace shared
+    # by all tasks within the group.
+    network {
+      # "mode" is the CNI plugin used to configure the network namespace.
+      # see the documentation for CNI plugins at:
+      #
+      #     https://github.com/containernetworking/plugins
+      #
+      mode = "bridge"
+
+      # The service we define for this group is accessible only via
+      # Consul Connect, so we do not define ports in its network.
+      # port "http" {
+      #   to = "8080"
+      # }
+    }
+    # The "service" stanza enables Consul Connect.
+    service {
+      name = "count-api"
+
+      # The port in the service stanza is the port the service listens on.
+      # The Envoy proxy will automatically route traffic to that port
+      # inside the network namespace. If the application binds to localhost
+      # on this port, the task needs no additional network configuration.
+      port = "9001"
+
+      # The "check" stanza specifies a health check associated with the service.
+      # This can be specified multiple times to define multiple checks for the
+      # service. Note that checks run inside the task indicated by the "task"
+      # field.
+      #
+      # check {
+      #   name     = "alive"
+      #   type     = "tcp"
+      #   task     = "api"
+      #   interval = "10s"
+      #   timeout  = "2s"
+      # }
+
+      connect {
+        # The "sidecar_service" stanza configures the Envoy sidecar admission
+        # controller. For each task group with a sidecar_service, Nomad  will
+        # inject an Envoy task into the task group. A group network will be
+        # required and a dynamic port will be registered for remote services
+        # to connect to Envoy with the name `connect-proxy-<service>`.
+        #
+        # By default, Envoy will be run via its official upstream Docker image.
+        # The optional "task" field overrides the default Envoy task with a
```

Comment:
This is no longer the case. We're working on a different method of overriding the Envoy task

Reply:
Is that https://github.com/hashicorp/nomad/issues/6040? Maybe we should wait on wrapping up this PR until that work has been solidified?

Reply:
No I think we can continue. Its covered it in the comment about referring to the task in `sidecar_service`